c.pod_design.md

```diff
@@ -201,6 +201,10 @@ kubectl annotate po nginx{1..3} description-
 <p>
 
 ```bash
+kubectl delete po --all
```

Comment:
this will delete all Pods on the default namespace. In this case it will work but in other cases (where there are other Pods in the cluster), this operation might be dangerous.